content/ytt/docs/latest/file-marks.md

```diff
@@ -65,15 +65,19 @@ Change type of file, affecting how `ytt` processes it.
 --file-mark '<path>:type=<file-type>'
 ```
 
-By default `file-type` is determined based on file extension. 
+By default `file-type` is determined based on file extension (and, as of v0.32.0, content).
 
 `file-type` can be: 
 - `yaml-template` (ext: `.yml`) — parsed as a YAML document containing `ytt` templating.
-- `yaml-plain` — parsed as a simple YAML document (not processed for templating).
+- `yaml-plain` (ext: `.yml`)  — parsed as a simple YAML document (not processed for templating).
```

Comment:
Should we specify .yaml as an extension as well, or are we taking an intentional stance?

Reply:
Yes, great point: we are accepting both. Best that the docs reflect that. 👍🏼